docker-compose.yml

```diff
@@ -242,14 +245,17 @@ services:
         tag: "desec/rabbitmq"
     restart: unless-stopped
 
-  celery:
+  celery-email:
     build: api
     image: desec/dedyn-api:latest
     init: true
-    command: celery -A api worker -l info --uid nobody --gid nogroup
+    command: celery -A api worker -E -Q email_slow_lane,email_fast_lane,email_immediate_lane -c 8 -n email -l info --uid nobody --gid nogroup
```

Comment:
removed `-E`. As for concurrency, we have i/o-bound tasks, which we can handle many in parallel, even if we have few cores. Fixing the number of threads to equal the number of cores makes sense when you got cpu-bound tasks that all eat up everything one core can provide

Reply:
I am not following: We have 8 cores in production, and you're fixing it to 8 child processes. If we did not do that, we'd have 16 child processes in case we upgrade to a machine with 16 cores, while with the current implementation, we'll stick to 8 child processes. Given that that we're i/o-bound, why should we fix it at such low number? (I would understand if you had fixed it at a higher number than we have cores -- but fixing it at exactly that number currently does nothing, but may limit us in the future.)

Reply:
You're right, 8 is essentially a function of "how much traffic do I except" and not of "how many cores do we have". If we had only one core, I'd still use 8 (or 16 or whatever). So declaring the attribute was to make this code meaningful for deployments with few cores (or the result of my ignorance as to how many cores we have). :blush:

Reply:
Closing this as if we ever send more emails than 8 processes can handle, we need to revisit email sending anyways.